partner-portal/src/UI/src/app/app.routes.ts

```diff
@@ -0,0 +1,17 @@
+import { Routes } from '@angular/router';
+import { RemedialComponent } from '@app/remedial/remedial.component';
+import { SearchComponent } from '@app/search/search.component';
+
+export const ROUTE_DENIED = 'denied';
```

Comment:
does partner portal have a denied page?

Reply:
For now we don't have the denied Route but might need that. For now i will remove that and add that when it is required.